.github/workflows/release-please.yml

```diff
@@ -9,13 +9,12 @@ jobs:
   release-please:
     outputs:
       release_created:  ${{ steps.release.outputs.release_created }}
-    permissions:
-      contents: write # to create release commit (googleapis/release-please-action)
-      pull-requests: write # to create release PR (googleapis/release-please-action)
     runs-on: ubuntu-latest
     steps:
       - uses: googleapis/release-please-action@v4
         id: release
+        with:
+          token: ${{ secrets.GH_USER_TOKEN }}
```

Comment:
```suggestion
```

I think we should continue to use the workflow permissions model to lessen the need for long lived tokens. Obviously this PR will need one to update `gyp-next` but I think it's better to create a single token whose only responsibility is creating the `gyp-next` PR.